test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpTraceExporterTests.cs

```diff
@@ -551,6 +551,32 @@ public void ToOtlpSpanActivityStatusTakesPrecedenceOverStatusTagsWhenActivitySta
             Assert.Equal(StatusDescriptionOnError, otlpSpan.Status.Message);
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void ToOtlpSpanTraceStateTest(bool traceStateWasSet)
```

Comment:
do we have existing tests for checking other things from activity context? like spanid,traceid etc? we could just modify it instead of adding dedicated tests.

Reply:
I can add traceState assertions to `ToOtlpSpanTest`.
Personally, I prefer to keep unit tests small.